themes/qgrid/source/js/menu.js

```diff
@@ -0,0 +1,49 @@
+function toggleDisplay(e) {
+	e.stopImmediatePropagation();
+	var nav = document.getElementById("nav");
+	var overlay = document.getElementById("overlay");
+	if (nav.style.display === 'block') {
+		nav.style.display = 'none';
+		overlay.style.display = 'none';
+	} else {
+		nav.style.display = 'block';
+		overlay.style.display = 'block';
+	}
+}
+
+function filter() {
+	var input = document.getElementById('search');
+	var filter = input.value.toLowerCase();
+	var nav = document.getElementById("nav");
+	var li = nav.getElementsByTagName('li');
+
+	for (var i = 0; i < li.length; i++) {
+		var a = li[i].getElementsByTagName("a")[0];
+		if (a.innerHTML.toLowerCase().indexOf(filter) > -1) {
+			li[i].style.display = "";
+		} else {
+			li[i].style.display = "none";
+		}
+	}
+}
+
+function init() {
+
+	var triggerBtns = document.getElementsByClassName("nav-trigger");
```

Comment:
i believe we are going to support only 1 trigger button :)

Reply:
Yes, but I had to place them in different templates (for nav and for header). They won't be visible togeather at the same time

Reply:
ok this is make sence can you add a small comment about this, please